openmdao/docs/usr-guide/tutorials/recording.rst

```diff
@@ -85,7 +85,7 @@ optimizer progresses.
 ::
 
     recorder = SqliteRecorder('paraboloid')
-    driver.add_recorder(recorder)
+    top.driver.add_recorder(recorder)
```

Comment:
Good idea. I was wondering why we weren't doing doctests on these.


From: Kenneth Moore <notifications@github.com<mailto:notifications@github.com>>
Reply-To: OpenMDAO/OpenMDAO <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, October 14, 2015 9:43 AM
To: OpenMDAO/OpenMDAO <OpenMDAO@noreply.github.com<mailto:OpenMDAO@noreply.github.com>>
Cc: Herbert Schilling <hschilling@nasa.gov<mailto:hschilling@nasa.gov>>
Subject: Re: [OpenMDAO] Prepended top. for some lines of code where it was missing (#347)


In openmdao/docs/usr-guide/tutorials/recording.rst<https://github.com/OpenMDAO/OpenMDAO/pull/347#discussion_r41994032>:

> @@ -85,7 +85,7 @@ optimizer progresses.
>  ::
>
>      recorder = SqliteRecorder('paraboloid')
> -    driver.add_recorder(recorder)
> +    top.driver.add_recorder(recorder)


Might want to put a doctest directive on these sections so that they run. that would prevent syntax errors. I'd recommend just doing it on the instantiation lines, but don't make the model run inside a doctest, since I don't know where the sql file will go or how to clean it up.

—
Reply to this email directly or view it on GitHub<https://github.com/OpenMDAO/OpenMDAO/pull/347/files#r41994032>.